tools/scripts/build_projects.py

```diff
@@ -112,7 +112,7 @@ def run_cmd_stm(cmd):
 		ERR = 1
 		return err
 	err = os.system(cmd + ' >> %s 2>&1' % log_file)
-	if err != 0 and err != 31744 and tmp[0] != 'cp':
```

Comment:
this looks like a revert of https://github.com/analogdevicesinc/no-OS/commit/4048b1ea4249104937c319b5efaefb86a8645cb5 ? Now there is another error treated (`124`). Maybe add separate commit explaining why and what you did. It is important to keep good track of these infrastructure changes.